src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -227,7 +231,9 @@ export default function (props: BloodPressureVisualizationProps) {
         return <div className="mdhui-blood-pressure-metrics-block">
             <div className="mdhui-blood-pressure-metric-value">{value ?? "--"} <span className="mdhui-blood-pressure-units">mm HG</span></div>
             <div className="mdhui-blood-pressure-metric-description">{description}</div>
-            <div className={alertClass}>{getAlertPrompt(alert) ?? language('no-data-yet')}</div>
+            {!props.suppressGuidance &&
+                <div className={alertClass}>{getAlertPrompt(alert) ?? language('no-data-yet')}</div>
```

Comment:
This is pre-existing, but I think we can drop the `?? language('no-data-yet')` from this.  It looks like the `getAlertPrompt(...)` function never returns `null` or `undefined`.